ocamldoc/Makefile.docfiles

```diff
@@ -42,7 +42,7 @@ DOC_COMPILERLIBS_INCLUDES = $(addprefix -I $(SRC)/, $(DOC_COMPILERLIBS_DIRS))
 DOC_ALL_INCLUDES = $(DOC_STDLIB_INCLUDES) $(DOC_COMPILERLIBS_INCLUDES)
 
 STDLIB_MOD_WP = $(filter-out stdlib__pervasives, $(STDLIB_MODULES))
-STDLIB_MLI0 = $(SRC)/stdlib/pervasives.ml $(STDLIB_MOD_WP:%=$(SRC)/stdlib/%.mli)
+STDLIB_MLI0 = $(STDLIB_MOD_WP:%=$(SRC)/stdlib/%.mli)
```

Comment:
Pervasives is still linked from the manual in http://caml.inria.fr/pub/docs/manual-ocaml/core.html. Just removing it creates a dead link.

Reply:
A hackish solution might be to generate `Pervasives` html file separately to get it out of the indexes.

Reply:
Would it be OK to just remove the link from library/core.etex? (leaving the text)

It's links to a copy of Stdlib without any documentation, which doesn't seem very useful.

Reply:
I support the removal of the "Pervasives" link and page.  What about a note that says "The Stdlib module used to be named Pervasives.  This usage is deprecated." ?

Reply:
Such a note is already present. I'm going to leave the mention of Pervasives and the deprecation note, but it won't be a link any more.

Reply:
Change in fb18d85

Reply:
I am ok with removing `Pervasives` since there is indeed a lot of breadcrumbs leading to the `Stdlib` module (Sorry I missed your answer last week).